tests/gce_test.go

```diff
@@ -138,6 +147,28 @@ func (p *process) Close() error {
 	return p.sess.Close()
 }
 
+func compileProxy() (binaryPath string, _ error) {
+	// Find the 'build.sh' script by looking for it in cwd, cwd/.., and cwd/../..
+	var buildSh string
+	for parents := 0; parents < 2; parents++ {
+		cur := filepath.Join(strings.Repeat(".."+string(filepath.Separator), parents), buildShLocation)
```

Comment:
Not a big deal, but how about having a slice for base path that starts as an empty slice and then appending .. to it with each loop iteration, that way you aren't doing your own filepath.Join via strings.Repeat?